### PR TITLE
Fix mixed case keys

### LINKED ISSUE #1777: Using caps in Keys causes issues
### Summary
It used to work.

### Steps To Reproduce
Create a new secret `t1` containing:
```
secret
User: test
```

Display it (with `safecontent` set to `true`):
```
$ gopass show t1
Secret: t1

User:
```

with safecontent set to `false` it works (but adds a trailing newline):
```
Secret: t1

secret
User: test

```

Furthermore the key `User` is not recognized:
```
$ gopass show t1 user

Error: key not found in entry
```
Both with `User` and with `user` as the queried keys and both with and without `safecontent` enabled

### Expected behavior
It should just not display the "secret" if safecontent is set (and the unsafe keys, if these are used)

### PR DESCRIPTION
**Pull request by AnomalRoil**
RELEASE_NOTES=[BUGFIX] fixed mixed case keys for key-value, all keys are lower case now

Fixes #1777

### CODE REVIEW

File: internal/action/show.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func (s *Action) showHandleOutput(ctx context.Context, name string, sec gopass.S
 		return nil
 	}
 
-	ctx = out.WithNewline(ctx, ctxutil.IsTerminal(ctx))
+	ctx = out.WithNewline(ctx, false && ctxutil.IsTerminal(ctx))
```
Comment (dominikschulz) [inline internal/action/show.go:161]: Why are you always disabling the newline output?
If that's correct you could remove the second part of the condition.

But I guess this is not intentional?

File: internal/action/show.go [hunk 2]
```diff
@@ -204,7 +204,7 @@ func (s *Action) showGetContent(ctx context.Context, sec gopass.Secret) (string,
 	// everything but the first line
 	if ctxutil.IsShowSafeContent(ctx) && !ctxutil.IsForce(ctx) {
 		var sb strings.Builder
-		for _, k := range sec.Keys() {
+		for l, k := range sec.Keys() {
```
Comment (dominikschulz) [inline internal/action/show.go:207]: Usually one would expect n or i for the counter. l as in lines?

File: internal/action/show.go [hunk 3]
```diff
@@ -218,7 +218,9 @@ func (s *Action) showGetContent(ctx context.Context, sec gopass.Secret) (string,
 				}
 				sb.WriteString(strings.Join(v, "\n"+k+": "))
 			}
-			sb.WriteString("\n")
+			if sec.Body() != "" || l < len(sec.Keys())-1 {
```
Comment (dominikschulz) [inline internal/action/show.go:221]: Ok, I get what you're doing but I think this is suble enough to ask for a short comment.

File: pkg/gopass/secrets/kv.go [hunk 4]
```diff
@@ -82,15 +82,20 @@ func (k *KV) Bytes() []byte {
 	buf := &bytes.Buffer{}
 	buf.WriteString(k.password)
 	buf.WriteString("\n")
-	for _, key := range k.Keys() {
+	for lk, key := range k.Keys() {
```
Comment (dominikschulz) [inline pkg/gopass/secrets/kv.go:85]: I expect this to be called `ik`

File: pkg/gopass/secrets/kv.go [hunk 5]
```diff
@@ -82,15 +82,20 @@ func (k *KV) Bytes() []byte {
...
 		sv, ok := k.data[key]
 		if !ok {
 			continue
 		}
-		for _, v := range sv {
+		for lv, v := range sv {
```
Comment (dominikschulz) [inline pkg/gopass/secrets/kv.go:90]: `iv`

File: pkg/gopass/secrets/kv.go [hunk 6]
```diff
@@ -82,15 +82,20 @@ func (k *KV) Bytes() []byte {
...
 			_, _ = buf.WriteString(key)
 			_, _ = buf.WriteString(": ")
 			_, _ = buf.WriteString(v)
+			if lv < len(sv)-1 {
+				_, _ = buf.WriteString("\n")
+			}
+		}
+		if k.body != "" || lk < len(k.Keys())-1 {
```
Comment (dominikschulz) [inline pkg/gopass/secrets/kv.go:98]: Short comment, please.

File: pkg/gopass/secrets/kv.go [hunk 7]
```diff
@@ -202,6 +207,8 @@ func ParseKV(in []byte) (*KV, error) {
 		for i, part := range parts {
 			parts[i] = strings.TrimSpace(part)
 		}
+		// we only store lower caps keys for KV
```
Comment (dominikschulz) [inline pkg/gopass/secrets/kv.go:210]: `lower case`, not `lower caps`?

**Review by dominikschulz — CHANGES_REQUESTED** [review]
Looks good, only some minor comments.

File: internal/action/show.go (hunk 1 above)
Comment (AnomalRoil) [reply internal/action/show.go:161]: Ups, leftover from debugging!

**Review by dominikschulz — APPROVED** [review]
LGTM

**Pull request merged**